recipes/dlib/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,8 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package LANGUAGES CXX)
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+cmake_minimum_required(VERSION 3.15)
+project(test_package CXX)
 
 find_package(dlib REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} dlib::dlib)
-set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+target_link_libraries(${PROJECT_NAME} PRIVATE dlib::dlib)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

Comment:
If it's a c++11 library, isn't the recipe missing a `validate()` method?

Reply:
Note: Do not copy cmake_template without verifying C++ required standard. 😝

Reply:
Fixed!